include/picongpu/particles/Particles.tpp

```diff
@@ -184,8 +184,12 @@ namespace picongpu
         else
             exchangeBytes = ExchangeMemCfg::BYTES_CORNER;
 
+        /* PIConGPU default case is running with 32bit (4byte) precision. In case double precision is used the particle
+         * buffers must be twice as large to avoid that a communication must be split into multiple sends.
+         */
+        double precisionMultiplier = static_cast<double>(sizeof(float_X) / 4u);
```

Comment:
```suggestion
        double precisionMultiplier = static_cast<double>(sizeof(float_X) / sizeof(float));
```

Does not change the logic, but is more description

Reply:
Or maybe even convert both sizeofs to `double` and then divide if we are very generic here

Reply:
I had this first too but switched and used `4u`. Maybe I will use what you suggested.

Reply:
> Or maybe even convert both sizeofs to double and then divide if we are very generic here

Will do it but only to be ready to use half-precision floats.